drivers/hmis/lib/form_data/assessments/base_assessment.json

```diff
@@ -389,12 +389,82 @@
             }
           ],
           "item": [
+            {
+              "type": "BOOLEAN",
+              "link_id": "4.02.3",
+              "hidden": true,
+              "field_name": "earned",
+              "autofill_values": [
+                {
+                  "value_boolean": true,
+                  "autofill_behavior": "ALL",
```

Comment:
Are these inverted?  Shouldn't it be `"autofill_behavior": "ANY",` here, and ALL below?